tools/bin/release_version.sh

```diff
@@ -40,7 +40,7 @@ echo "Bumped version from ${PREV_VERSION} to ${NEW_VERSION}"
 echo "Building and publishing version $NEW_VERSION for git revision $GIT_REVISION..."
 
 SUB_BUILD=PLATFORM ./gradlew clean composeBuild
-SUB_BUILD=PLATFORM ./gradlew clean publish
```

Comment:
@davinchia @ChristopheDuong this was causing the script to fail because docker compose build needs a file in `airbyte-server/build/config_init`. running the clean wipes it out. it's not actually obvious to me why we were cleaning before publish anyway. what was the reason for it?